Sources/SwiftDocCUtilities/ArgumentParsing/Subcommands/Convert.swift

```diff
@@ -167,6 +167,12 @@ extension Docc {
         @OptionGroup()
         public var sourceRepositoryArguments: SourceRepositoryArguments
 
+        /// A user-provided value that is true if experimental Doxygen support should be enabled.
+        ///
+        /// Defaults to false.
+        @Flag(help: "Parse a limited set of Doxygen commands as equivalent DocC markup")
```

Comment:
```suggestion
        @Flag(help: "Parse a limited set of Doxygen commands as equivalent Swift-DocC documentation markup")
```

Reply:
In the past we've marked these as hidden but I don't have a very strong opinion here either way.

```suggestion
        @Flag(help: .hidden)
```